website/static/js/profile.js

```diff
@@ -483,17 +483,34 @@ var SocialViewModel = function(urls, modes) {
     TrackedMixin.call(self);
 
     self.addons = ko.observableArray();
+    
+    self.profileWebsites = ko.observableArray();
+
+    self.profileWebsitesLength = ko.computed(function() {
+        return self.profileWebsites().length;
+    });
+
+    self.hasProfileWebsites = ko.pureComputed(function() {
```

Comment:
I tried this previously, and ran into some issues with the timing of knockout.  The check was not done at the appropriate time.